src/app/view/components/_common/site-selector/site-selector.component.ts

```diff
@@ -85,6 +85,19 @@ export class SiteSelectorComponent implements OnInit, OnChanges {
      */
     handleSitesRefresh(): void {
         this.paginationService.getCurrentPage().subscribe((items) => {
+            const sites: any[] = items.splice(0);
```

Comment:
since the interfase for SITE is outdated, the way to get filled the array with data is not specifying a type.